hs_core/views/__init__.py

```diff
@@ -154,9 +154,11 @@ def update_quota_usage(request, username):
     except User.DoesNotExist:
         return HttpResponseBadRequest('user to update quota for is not valid')
 
-    update_quota_usage_utility(user=user)
-
-    return HttpResponse('quota for user ' + user.username + ' has been updated', status=200)
+    ret_status = update_quota_usage_utility(username)
+    if ret_status:
+        return HttpResponse('quota for user ' + username + ' has been updated', status=200)
+    else:
+        return HttpResponse('quota for user ' + username + ' failed to update. Check logs for details.', status=500)
```

Comment:
I don't feel like 500 is the proper response.  That's really reserved for when we don't know what went wrong.   We do know what went wrong in this situation.  Either UserQuota doesn't exist for this user in django or irods doesn't have an avu for the user.  Is there a reason why it's not returned in the response?

Reply:
@sblack-usu I thought 500 was the response that captures the unexpected server-side issues, although I agree the response can include the specific error rather than a generic failed-to-update error. I can make the response error message more specific to what has really happened, but let me know what other response status code than 500 would be more appropriate here.

Reply:
Looking over status codes, I'm comfortable with 500 as long as we include the reason in the response.  I don't see any reason to limit it only to the logs.